contrib/nydusify/cmd/nydusify.go

```diff
@@ -409,10 +409,10 @@ func main() {
 					Usage:   "Path to the nydus-image binary, default to search in PATH",
 					EnvVars: []string{"NYDUS_IMAGE"},
 				},
-				&cli.BoolFlag{
+				&cli.StringFlag{
 					Name:    "output-json",
-					Value:   false,
-					Usage:   "Enable saving the metrics collected during conversion in JSON file output.json",
+					Value:   "",
+					Usage:   "File path to save the metrics collected during conversion in JSON format, for example: './metric.json'",
```

Comment:
'./output.json' should be better, we don't assume that the file only stores metrics for now.

Reply:
fxied